ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -506,5 +493,23 @@ router bgp {{ router_bgp.as }}
 {%             endif %}
       {{ aggregate_address_cli }}
 {%         endfor %}
+{%         for  address_family in router_bgp.vrfs[vrf].address_families | arista.avd.natural_sort %}
+      !
+      address-family {{ address_family }}
+{%             for neighbor in router_bgp.vrfs[vrf].address_families[address_family].neighbors | arista.avd.natural_sort %}
+{%                 set neighbor_cli = "neighbor " ~ neighbor %}
+{%                 if router_bgp.vrfs[vrf].address_families[address_family].neighbors[neighbor].activate is arista.avd.defined(true) %}
+{%                     set neighbor_cli = neighbor_cli ~ " activate" %}
```

Comment:
`neighbor x.x.x.x` with nothing else after it is not a valid CLI command. So we should check for activate and only print the full cli if it is set.
```
(config-router-bgp-vrf-hat)#neighbor 1.2.3.4 
% Incomplete command
```

Reply:
Fixed it for you :)